shared/actions/teams.js

```diff
@@ -79,9 +79,13 @@ function* joinTeam(_, action) {
     Saga.put(TeamsGen.createSetTeamJoinSuccess({success: false, teamname: ''})),
   ])
   try {
-    const result = yield* Saga.callPromise(RPCTypes.teamsTeamAcceptInviteOrRequestAccessRpcPromise, {
-      tokenOrName: teamname,
-    })
+    const result = yield* Saga.callPromise(
+      RPCTypes.teamsTeamAcceptInviteOrRequestAccessRpcPromise,
+      {
```

Comment:
i've been making objects w/ one key just go on a single line, easier to read i think but 🐑

Reply:
o hmm, I thought this was controlled by my formatter. I'll try to change it

Reply:
so prettier seems to have different rules given its input so if you do like
```
{a: 1}
``` 
it'll leave it
but if you do like
```
{a:
1}
```
it'll make it 3 lines